Tests/Shared.Specs/BasicEquivalencySpecs.cs

```diff
@@ -1918,6 +1918,18 @@ public void
             act.Should().Throw<XunitException>().Which.Message.Should().Contain("Field1").And.Contain("Property1");
         }
 
+        [Fact]
+        public void When_excluding_virtual_or_abstract_property_exclusion_works_properly()
```

Comment:
Don't you also need an example that uses that `VirtualProperty`?

Reply:
Done

Reply:
I meant another unit test. I mean, you exclude `VirtualProperty` and `DerivedProperty`, but your _actual_ and _expectation_ don't seem to set those properties.